webbpsf/webbpsf_core.py

```diff
@@ -115,6 +115,19 @@ class SpaceTelescopeInstrument(poppy.instrument.Instrument):
         The SAM code will in general be much faster than the FFT method, particularly for high oversampling.
 
     """
+    _detectors = {}
+    """
+    Dictionary mapping detector names to FieldDependentAberration optics.
```

Comment:
Needs update - in the case of JWST instruments it's a dictionary mapping detector names to SIAF entries.